job_definitions/generate_upload_and_notify_counterpart_signature_pages.yml

```diff
@@ -21,60 +16,60 @@
           <li>Upload the generated files to S3 and set the path in the relevant FrameworkAgreement record</li>
           <li>Send an email to all active users on the supplier account that the counterpart file has been uploaded</li>
         </ol>
+    project-type: pipeline
     concurrent: false
-    triggers:
-      - timed: 'H 1 * * *'
     properties:
       - build-discarder:
           days-to-keep: 7
           artifact-days-to-keep: 7
+    parameters:
+      - choice:
+          name: ENVIRONMENT
+          choices:
+            - preview
+            - staging
+            - production
+      - string:
+          name: FRAMEWORK
     dsl: |
       node {
           try {
-            stage('Prepare') {
-              dir('scripts/generated-countersignature-pdfs') {
-                deleteDir()
```

Comment:
Are we no longer explicitly cleaning up this folder? We have a new data folder cleaned & created below - is that just for cloning repos? Where do the PDFs go?

Reply:
PDFs go to `data/generated-countersignature-pdfs`, so it gets cleaned up by the `rm -rf` below

Reply:
Aren't we deleting `scripts/generated-countersignature-pdfs` in the `Cleanup` stage?

Reply:
Ah good spot!!